### ISSUE TITLE
MultiProgrammLanguage - Go Client

### ISSUE THREAD

**Issue by NickNYU** [sofastack/sofa-registry#171]

**Comment by ZLBer** [comment]
I'm interested in this .But  I found that the Java client uses  sofa-bolt to communicate with sofa-registry, sofa-bolt has no Go version, and as far as I know, there is no HTTP communication. How should Go communicate with sofa-registry ？

**Comment by NickNYU** [comment]
assigned to you, @ZLBer 

you are well welcomed for joining SOFARegistry

Any questions, problems we could talk in this channel

Thanks

**Comment by ZLBer** [comment]
the problem is above, I can't do it without this.

**Comment by seeflood** [comment]
> the problem is above, I can't do it without this.

Yep,it seems that this task is blocked,since we don't have a bolt-go library  :<

I will publish a bolt-go task and wait for someone who is interested at that.

But for now we have to suspend this task
